OWNERS

```diff
@@ -0,0 +1,79 @@
+// For an explanation of the OWNERS rules and syntax, see:
+// https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example
+
+{
+  rules: [
+    {
+      owners: [
+        {
+          name: 'cramforce',
+          requestReviews: false
+        },
+        {
+          name: 'dvoytenko',
+          requestReviews: false
+        },
+        {
+          name: 'jridgewell',
+          requestReviews: false
+        }
+      ]
+    },
+
+    {
+      pattern: '**/validator-*.{protoascii,html,out}',
+      owners: [
+        {
+          name: 'ampproject/wg-caching',
+          notify: true
+        }
+      ]
+    },
+
+    {
+      pattern: '*.md',
+      owners: [
+        { name: 'ampproject/wg-outreach' }
+      ]
+    },
+
+    {
+      pattern: '{.*,babel.config.js,gulpfile.js}',
+      owners: [
+        { name: 'ampproject/wg-infra' }
```

Comment:
I noticed that the new json5 files have initially been auto-formatted w.r.t. indentation, new lines, etc. As people manually edit `OWNERS` files from here on, they could break the formatting convention and / or introduce parsing errors. Would it be possible to have `prettier` auto-format these files like other `.js` files?

Reply:
Filed tracking issue #24936 to address in follow-up PR (since this PR is already massive)

Reply:
All `OWNERS` files were auto formatted in #25085.